src/include/parser/create_statement.h

```diff
@@ -174,6 +181,8 @@ struct ColumnDefinition {
   std::vector<char*>* foreign_key_source = nullptr;
   std::vector<char*>* foreign_key_sink = nullptr;
 
+  std::vector<char *>* multi_unique_cols = nullptr;
+
   char* foreign_key_table_name = nullptr;
   FKConstrActionType foreign_key_delete_action;
```

Comment:
foreign_key_delete_action, foreign_key_update_action, foreign_key_match_type are not initialized in the constructor.